plugins/MauticFocusBundle/Include/lessc.inc.php

```diff
@@ -716,7 +716,7 @@ protected function zipSetArgs($args, $orderedValues, $keywordValues)
 
         // check for a rest
         $last = end($args);
-        if ('rest' == $last[0]) {
+        if (is_array($last) && count($last) && 'rest' == $last[0]) {
```

Comment:
I would suggest this:
```suggestion
        if (isset($last[0]) && 'rest' == $last[0]) {
```
As the first 2 checks do not ensure that the array has a key of 0. What do you think?

Reply:
While I agree with you and solution with isset is better (thanks for pointing it out). I think I had version of this before and it did not really cover all the cases as sometimes $last does not appear to be array. I think we still need to check if $last is array. Using your suggestion and my note in this comment I would recommend the following change.

```suggestion
        if (is_array($last) && isset($last[0]) && 'rest' == $last[0]) {
```

What do you think?